conf.py

```diff
@@ -1,10 +1,112 @@
 """
 Configuration for building the website with Sphinx.
 """
-import sys
-import os
+import re
 import datetime
 import subprocess
+from requests import get
+from pathlib import Path
+
+
+AUTHORS_BASE_URL = "https://raw.githubusercontent.com/fatiando/{}/{}/AUTHORS.md"
+AUTHOR_HTML_CARD = """
+<div class="col-6 col-md-3 col-sm-4">
+  <div class="card">
+    <img
+        class="card-img-top"
+        src="{avatar_url}"
+        alt="Profile picture of {full_name}"
+    >
+    <div class="card-body">
+      <h4 class="card-title fs-5">
```

Comment:
```suggestion
      <h4 class="card-title fs-6">
```

Again, smaller size to better fit the smaller columns.